webserver/cWebem.cpp

```diff
@@ -1886,16 +1804,16 @@ namespace http {
 
 		bool cWebemRequestHandler::CheckAuthentication(WebEmSession & session, const request& req, reply& rep)
 		{
-			bool bTrustedNetwork = false;
-
 			session.rights = -1; // no rights
```

Comment:
Maybe we should create a new (first) enum entry in **_eUserRights** for this and use this instead of -1
Should this be the default in the object's constructor? If yes, then no need to initialize it here

Reply:
done